### ISSUE TITLE
Version Restrictions

### ISSUE THREAD

**Issue by Kirksten3** [supergoodsystems/supergood-py#41]
Hey all! Very interested in dropping this in on one of our projects. Unfortunately, we seem to have hit a road block with some of the version restrictions on this package.

Is it possible to bump the dependencies to something like:
```
dependencies = [
    "aiohttp",
    "httpx",
    "jsonpickle",
    "pydash>=7.0.1,<9",
    "python-dotenv>=1.0.0,<1.1.0",
    "requests",
    "tldextract>=5",
    "urllib3",
]
```
We have conflicts on both `python-dotenv` and `pydash`. From a cursory check neither appear to have breaking changes. `pydash` had no changes to `set_` or `get`.

Happy to test & submit a PR if that's easier, just let me know.

**Comment by stevenbunting** [comment]
Hey, thanks for the submission!
Don't forsee any issues, but I'll build a test branch with these dependencies and run it against our internal tests this weekend. Appreciate the level of detail you provided.

**Comment by stevenbunting** [comment]
Apologies for the delay, but I've just released version 2.1.1 which includes this dependency relaxation. Let us know if you run into any other issues!

**Comment by Kirksten3** [comment]
Hey no worries, and thanks! I'll give this another try tonight
